### DISCUSSION TITLE
Full height docks

### DISCUSSION THREAD

**Discussion by Dunravin** [obsproject/obs-studio#9886] [Development Questions]
This is a welcomed feature but its not working entirely right. With Full Height Docks enabled its forcing the window size to spill over on to my main monitor, I can resize wider but not smaller and its not possible to maximise the OBS window to my monitor.

It would be beneficial to also be to selectively control which dock gets a full height. For example its really only needed for the chat dock window and if that were possible it might mitigate the first window sizing issue.

[Edit] Locking the docks kind of works for maximising the OBS window but there's some strange behaviour going on when attempting to resize the panels and docks

**Comment by Fenrirthviti** [comment]
This sounds more likely to be a support request, and we are not currently accepting support requests on GitHub Discussions. Make sure you read any [pinned posts](https://github.com/obsproject/obs-studio/discussions/5206) on discussion boards before opening a new discussion. Please use our [forums](https://obsproject.com/forum/) or [Discord](https://obsproject.com/discord/) for further assistance.

Thank you!
